### ISSUE TITLE
Keybindings for scrolling line by line

### ISSUE THREAD

**Issue by Riyyi** [tomscii/zutty#113]
My mouse scroll wheel is broken, so I'd like a keyboard option to scroll line by line. I don't know if this is possible yet.
Currently, I'm using `urxvt` and have this in my Xresources:
```yaml
URxvt.keysym.Shift-Up: command:\033]720;1\007
URxvt.keysym.Shift-Down: command:\033]721;1\007
```
Which maps shift+up and shift+down to scroll 1 line.

**Comment by tomscii** [comment]
> I don't know if this is possible yet.

Zutty is documented - what you are stating here is that you haven't bothered to look at it. (But you are asking me to care about your broken hardware.)

Of course it is possible, by applying a small and fairly obvious modification to the source code. Coming up with that change is an exercise left to you (if you have time to deal with a broken scroll wheel...)

Other than that, I am not considering such a feature - Zutty is minimalistic, and a generally extensible keyboard binding mechanism so anyone can do anything is simply out of scope.

Note that you already have a way to exercise scrollback via the keyboard (Shift+PageUp/PageDown), just not line by line.

**Comment by Riyyi** [comment]
>Zutty is documented - what you are stating here is that you haven't bothered to look at it.

Maybe I worded it poorly, I had looked at the documentation but I was unsure if I misunderstood or missed this possibility.

>Of course it is possible, by applying a small and fairly obvious modification to the source code.

This was my assumption, but seeing the minimalistic nature of this project, my intent of this issue was to ask if you would consider something like this upstream. I'm not interested in maintaining a fork.

Guess I'll stick to `urxvt` until Wayland becomes usable.
